src/datepicker/datepicker-input.ts

```diff
@@ -185,6 +185,13 @@ export class NgbInputDatepicker implements OnChanges,
    */
   @Output() navigate = new EventEmitter<NgbDatepickerNavigateEvent>();
 
+  /**
+   * An event fired after datepicker closing the datepicker.
+   *
+   * @memberof NgbInputDatepicker
```

Comment:
We usually don't use this kind of jsdoc comments. Could you please remove it ?

Reply:
Thanks for advice, removed.